mgmt/rest/server.go

```diff
@@ -492,6 +507,41 @@ func (s *Server) addRoutes() {
 		s.r.GET("/v1/tribe/members", s.getMembers)
 		s.r.GET("/v1/tribe/member/:name", s.getMember)
 	}
+
+	// profiling tools routes
+	if s.pprof {
```

Comment:
are you sure that you want to dynamically change it when snapd is already running? Will it need to restart snapd by doing this way?